src/Tables.jl

```diff
@@ -565,6 +565,29 @@ struct Partitioner{T}
     x::T
 end
 
+"""
+    getrows(x, inds; view=nothing)
```

Comment:
```suggestion
    Tables.getrows(x, inds; view=nothing)
```
We include the `Tables` prefix since this won't be exported.